src/main/java/mate/academy/db/DatabaseInitializer.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.db;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class DatabaseInitializer {
+    private static final String FILE_PATH = "src/main/resources/init_db.sql";
+    private static final String SQL_SCRIPT = """
+        CREATE TABLE `books` (
+            `id` INT NOT NULL AUTO_INCREMENT,
+            `title` VARCHAR(255) NOT NULL,
+            `price` DECIMAL(10, 2) NOT NULL
```

Comment:
There is a missing comma at the end of this line. Add a comma to separate the 'price' column definition from the 'PRIMARY KEY' constraint.